ecl/data_source_ecl_vna_appliance_plan_v1_test.go

```diff
@@ -0,0 +1,127 @@
+package ecl
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/terraform"
+)
+
+const testAccVNAV1AppliancePlanDataSourcePlanName = "vSRX_20.4R2_2CPU_4GB_8IF_STD"
+
+func TestAccVNAV1AppliancePlanDataSource_basic(t *testing.T) {
+	resource.Test(t, resource.TestCase{
+		PreCheck:  func() { testAccPreCheck(t) },
+		Providers: testAccProviders,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccVNAV1AppliancePlanDataSourceBasic,
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheckVNAV1AppliancePlanDataSourceID("data.ecl_vna_appliance_plan_v1.appliance_plan_1"),
+					resource.TestCheckResourceAttr(
+						"data.ecl_vna_appliance_plan_v1.appliance_plan_1", "name", testAccVNAV1AppliancePlanDataSourcePlanName),
+					resource.TestCheckResourceAttr(
+						"data.ecl_vna_appliance_plan_v1.appliance_plan_1", "description", testAccVNAV1AppliancePlanDataSourcePlanName),
+					resource.TestCheckResourceAttr(
+						"data.ecl_vna_appliance_plan_v1.appliance_plan_1", "appliance_type", "ECL::VirtualNetworkAppliance::VSRX"),
+					resource.TestCheckResourceAttr(
+						"data.ecl_vna_appliance_plan_v1.appliance_plan_1", "flavor", "2CPU-8GB"),
+					resource.TestCheckResourceAttr(
+						"data.ecl_vna_appliance_plan_v1.appliance_plan_1", "enabled", "true"),
+					resource.TestCheckResourceAttr(
+						"data.ecl_vna_appliance_plan_v1.appliance_plan_1", "licenses.#", "1"),
+					resource.TestCheckResourceAttr(
+						"data.ecl_vna_appliance_plan_v1.appliance_plan_1", "licenses.0.license_type", "STD"),
+					resource.TestCheckResourceAttr(
+						"data.ecl_vna_appliance_plan_v1.appliance_plan_1", "availability_zones.#", "1"),
```

Comment:
```suggestion
						"data.ecl_vna_appliance_plan_v1.appliance_plan_1", "availability_zones.#", "3"),
```
The value of lab3 is 3.